libafl_qemu/librasan/asan/src/hooks/aligned_alloc.rs

```diff
@@ -0,0 +1,38 @@
+use core::{
+    ffi::{c_char, c_void},
+    mem::size_of,
+    ptr::null_mut,
+};
+
+use log::trace;
+
+use crate::{GuestAddr, asan_alloc, asan_panic, size_t};
+
+/// # Safety
+/// See man pages
+#[unsafe(export_name = "patch_aligned_alloc")]
+pub unsafe extern "C" fn aligned_alloc(alignment: size_t, size: size_t) -> *mut c_void {
+    unsafe {
+        trace!(
```

Comment:
I think we usually keep `log::trace!`

Reply:
(But also doesn't matter much)

Reply:
Also, if you add trace logs, you probably want to compile them out for release (IIRC the log crate has a feature flag to do that)

Reply:
Done. Trace is higher verbosity than debug, so should get compiled out too.